docs/faq.rst

```diff
@@ -808,7 +807,7 @@ There are some caveats:
 - If the repository is in "keyfile" encryption mode, the keyfile must
   exist locally or it must be manually moved after performing the upgrade:
 
-  1. Locate the repository ID, contained in the ``config`` file in the repository.
+  1. Get the repository ID with ``borg config /path/to/repo id``.
```

Comment:
hmm, we are dealing with an attic repo here, so guess we rather don't use `borg config` on it.

Reply:
hmm, no, it is after upgrade.